Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,26 @@
+Tool NuGet package format
+-------------------------------------------
+
+Use `dotnet pack` with `PackAsTool` property is the most straight forward way. However, if your build process is highly customized and `dotnet publish` cannot create a proper distribution for you. You can create tool NuGet package using nuspec and explicitly place assets in NuGet package under the following rules.
+
+- Tool NuGet package does not have any other folder under root like _/lib_, _/content_, it only has _/tools_ folder.
+- Under _/tools_ folder, the subfolders are structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 and portable should be under _tools/netcoreapp2.1/any_. Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets.

```

Comment:
- Under _/tools_ folder, the subfolders must be structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 that are portable across platforms should be in the folder _tools/netcoreapp2.1/any_. 

I do not understand what you are trying to say in the last sentence:

Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets.

Reply:
I try to make a definition 
per TFM-RID asset = assets under every tools/target framework short version/RID combination